xds/src/main/java/io/grpc/xds/XdsNameResolver.java

```diff
@@ -983,12 +971,19 @@ public void onResourceDoesNotExist(final String resourceName) {
   private static class RoutingConfig {
     private final long fallbackTimeoutNano;
     final ImmutableList<RouteData> routes;
-
-    private static final RoutingConfig EMPTY = new RoutingConfig(0, ImmutableList.of());
+    final Status status;
```

Comment:
nit: perhaps update the variable name to `errorStatus` to convey better intent (status is supposed to be `null` if the `RoutingConfig` is OK